.teamcity/components/settings.kt

```diff
@@ -0,0 +1,37 @@
+// specifies the default hour (UTC) at which tests should be triggered, if enabled
+var defaultStartHour = 0
+
+// specifies the default level of parallelism per-service-package
+var defaultParallelism = 10
+
+var locations = mapOf(
+        "public" to LocationConfiguration("westeurope", "eastus2", "francecentral", false),
+        "germany" to LocationConfiguration("germanynortheast", "germanycentral", "", false)
+)
+
+// specifies the list of Azure Environments where tests should be run nightly
+var runNightly = mapOf(
+        "public" to true
+)
+
+// specifies a list of services which should be run with a custom test configuration
+var serviceTestConfigurationOverrides = mapOf(
```

Comment:
presumably this list is going to want more tweaking based on trial/error - but that's outside the scope of this PR